tools/clang/test/HLSLFileCheck/hlsl/objects/HitObject/hitobject_make.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: %dxc -T lib_6_9 -E main %s | FileCheck %s
```

Comment:
All these tests currently under `HLSLFileCheck` can be moved under the `CodeGenDXIL` path to make them ordinary lit shell tests.  This is something we are generally advising for new tests that aren't required to run under this path due to special tooling only available here still.

When doing so, you need to add `// REQUIRES: dxil-1-9`

```suggestion
// RUN: %dxc -T lib_6_9 -E main %s | FileCheck %s
// REQUIRES: dxil-1-9
```